app/src/main/kotlin/com/simplemobiletools/dialer/activities/DialpadActivity.kt

```diff
@@ -49,45 +50,51 @@ class DialpadActivity : SimpleActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_dialpad)
+        setContentView(dialpadActivityBinding.root)
         hasRussianLocale = Locale.getDefault().language == "ru"
 
-        updateMaterialActivityViews(dialpad_coordinator, dialpad_holder, useTransparentNavigation = true, useTopSearchMenu = false)
-        setupMaterialScrollListener(dialpad_list, dialpad_toolbar)
+        updateMaterialActivityViews(
+            dialpadActivityBinding.dialpadCoordinator,
+            dialpadActivityBinding.dialpadHolder,
+            useTransparentNavigation = true,
+            useTopSearchMenu = false
+        )
+        setupMaterialScrollListener(dialpadActivityBinding.dialpadList, dialpadActivityBinding.dialpadToolbar)
         updateNavigationBarColor(getProperBackgroundColor())
 
         if (checkAppSideloading()) {
             return
         }
 
         if (config.hideDialpadNumbers) {
-            dialpad_1_holder.isVisible = false
-            dialpad_2_holder.isVisible = false
-            dialpad_3_holder.isVisible = false
-            dialpad_4_holder.isVisible = false
-            dialpad_5_holder.isVisible = false
-            dialpad_6_holder.isVisible = false
-            dialpad_7_holder.isVisible = false
-            dialpad_8_holder.isVisible = false
-            dialpad_9_holder.isVisible = false
-            dialpad_plus_holder.isVisible = true
-            dialpad_0_holder.visibility = View.INVISIBLE
+
+            dialpadActivityBinding.dialpadWrapper.dialpad1Holder.isVisible = false
```

Comment:
shorten these by using `dialpadActivityBinding.dialpadWrapper.apply {...}` please

Reply:
in the code below too